docs/modelserving/v1beta1/transformer/collocation/README.md

```diff
@@ -82,15 +92,15 @@ EOF
     predictor. The storage uri should be only present in this container. If it is specified in the transformer 
     container the isvc creation will fail.
 
-!!! Note
-    Currently, The collocation support is limited to the custom container spec for kserve model container.
-
 !!! Note
     In Serverless mode, Specifying ports for predictor will result in isvc creation failure as specifying multiple ports 
     is not supported by knative. Due to this limitation predictor cannot be exposed to the outside cluster. 
     For more info see, [knative discussion on multiple ports](https://github.com/knative/serving/issues/8471).
 
-## Check InferenceService status
+!!! Tip
+    Check the [Transformer documentation](../torchserve_image_transformer/#transformer-specific-commandline-arguments) for list of arguments that can be passed to the transformer container.
```

Comment:
We will need to start removing torch serve from the example as the project has limited maintenances.